packages/graphback/src/buildGraphbackAPI.ts

```diff
@@ -89,7 +94,9 @@ export function buildGraphbackAPI(model: string | GraphQLSchema, config: Graphba
   const serviceCreator = config.serviceCreator || createCRUDService({ pubSub: new PubSub() })
 
   const services = createServices(models, serviceCreator, config.dataProviderCreator)
-
+  const contextCreator = () => {
```

Comment:
Where is this getting overriden with the custom one from `GraphbackAPIConfig`?

Reply:
Done!